historical_system_profiles/views/v0.py

```diff
@@ -16,14 +20,60 @@ def get_pits_by_ids(profile_ids):
     """
     return a list of historical system profiles for the given profile IDs
     """
-    if profile_ids == ["100"]:
-        return mocks.MOCK_PROFILE_RESPONSE
-    else:
-        return {"results": []}
+    account_number = view_helpers.get_account_number(request)
+
+    query = HistoricalSystemProfile.query.filter(
+        HistoricalSystemProfile.account == account_number,
+        HistoricalSystemProfile.id.in_(profile_ids),
+    )
+
+    query_results = query.all()
+
+    result = query_results[0]
+    result.system_profile["id"] = profile_ids[0]
+    result.id = profile_ids[0]
+
+    return {"data": [result.to_json()]}
 
 
 def get_pits_by_inventory_id(inventory_ids):
     """
     return a list of historical system profiles for a given inventory id
     """
-    return mocks.MOCK_INV_RESPONSE
+    account_number = view_helpers.get_account_number(request)
+
+    query = HistoricalSystemProfile.query.filter(
+        HistoricalSystemProfile.account == account_number,
+        HistoricalSystemProfile.inventory_id.in_(inventory_ids),
+    )
+
+    query_results = query.all()
+    results = {
+        "inventory_uuid": inventory_ids[0],
+        "display_name": query_results[0].system_profile["display_name"],
+    }
+
+    profiles = [{"created": p.created_on, "id": p.id} for p in query_results]
```

Comment:
Is this querying based on creation of the profile in this service or the sync time of the profile itself?

Reply:
I believe it's creation of the profile, which is incorrect